### PR TITLE
docs: Updates to Tooltip and DropDown

### PR DESCRIPTION
**Pull request by zero0Halo**
docs(PositionGrid): new component, helps illustrate the positions of tooltip/dropdown

docs(ApiTable): new component to hopefully replace straight html tables being used for api section

docs(tooltip): Updated API table with more consistent wording/formatting

chore: prettier

chore: updated to v0.0.1-beta.18

fix(Tooltip): renamed wrap to wrapContent as wrap is a reserved html attribute

fix(DropDownExample): changed position from object to string, 'bottom'

fix(ColorBlurb): renamed file ColorBlurb to ColorBlurb.component

docs(DropDown): Added the PositionGrid component

docs(DropDownExample): added in all props from DropDown

docs(dropdown): Added ApiTable component, added in new props

**Before submitting a pull request,** please make sure the following is done:

I have done **all** of the following:

- [ ] Added a top level class to all my components `'.anchor-[COMPONENT NAME]'`.
- [x] Used [conventional commits](https://www.conventionalcommits.org) for all work.
- [ ] Tested my solution on Mobile & Tablet.
- [ ] Wrote [unit tests](https://jestjs.io/docs/en/getting-started) for states and all behavior (`npm test`) and passed coverage thresholds.
- [x] Updated snapshots for all permutations (`npm test -- -u`).
- [ ] Accounted for hover, focus, blur, visited, & error states because they are not edge cases.
- [ ] Created TODOs for known edge cases.
- [x] Documented all of my changes (inline & doc site).
- [ ] Made sure that all accessibility errors are resolved.
- [ ] Added [stories](https://storybook.js.org/docs/basics/introduction/) with knobs for all possible configurations.
- [x] De-linted and ran [prettier](https://github.com/prettier/prettier) (`npm run pretty`) on my code.
- [ ] Added name to OWNERS file for all new components
- [ ] If adding a new component, add its export to the rollup config
- [x] package.json version is bumped (if necessary)

---------
**Outline your feature or bug-fix below**

- Updates documentation on `DropDown`
- Updates documentation on `Tooltip`
- Updates Anchor to v0.0.1-beta.18
- Adds `ApiTable` component to eventually replace semantic table tags and related css
- Adds `PositionGrid` component to demonstrate absolute positioning in Anchor

### CODE REVIEW

File: docs/src/pages/components/dropdown.mdx [hunk 1]
```diff
@@ -91,36 +114,101 @@ together with `DropDown`!
...
+        property: 'showArrow',
+        description: `Whether or not the arrow should be displayed.`,
+        type: 'boolean',
+        default: 'true',
+    },
+    {
+        property: 'spacing',
+        description: `The distance the <pre>DropDown</pre> will be from the contained element. Any
+            valid CSS unit (<pre>em</pre>, <pre>px</pre>, etc.) is accepted. Note that this prop
+            does nothing if <pre>showArrow</pre> is set to <pre>false</pre>`,
```
Comment (brandonlilly) [inline docs/src/pages/components/dropdown.mdx]: Not quite right. It still does something if `showArrow` is `false`. The difference is that the default behavior is different if `showArrow` is `true`. i.e. The Default value is `0` if false and `0.75rem` if showArrow is true.

File: docs/src/pages/components/dropdown.mdx [hunk 2]
```diff
@@ -10,7 +16,14 @@ import { ComponentInfo, FormatTypes } from '../../components/Utils';
 
 <DropDownExample />
 
-<br />
+<br /><br />
+
+The `DropDown` has 12 possible placements via the `position` prop. These placements are based on the
+motion and direction of the content and less about exact positioning. The diagram below illustrates:
+
+<PositionGrid />
```
Comment (brandonlilly) [inline docs/src/pages/components/dropdown.mdx:24]: Love it

File: docs/src/components/Utils/ApiTable.component.tsx [hunk 3]
```diff
@@ -0,0 +1,161 @@
...
+interface ApiTableProps {
+    data: [];
+}
+
+interface RowData {
+    property: string;
+    /* There are several utility components that could be passed as a description in addition to a
+    plain string. Also an option is to pass an array of values in order to combine both plain text
+    and components.  */
+    description: string | [] | React.ReactElement;
```
Comment (brandonlilly) [inline docs/src/components/Utils/ApiTable.component.tsx:111]: Eventually I think we should have some default `descriptions` based on the `property`. For the kind of props that are basically the same everywhere (e.g. `borderRadius`), to reduce the copy pasta.

Looks good tho

**Review by brandonlilly — CHANGES_REQUESTED** [review]

File: docs/src/components/Utils/ApiTable.component.tsx (hunk 3 above)
Comment (zero0Halo) [reply docs/src/components/Utils/ApiTable.component.tsx:111]: Absolutely agree. I started doing that with the `ColorBlurb` component and intend on expanding on that trend.

File: docs/src/pages/components/dropdown.mdx (hunk 1 above)
Comment (zero0Halo) [reply docs/src/pages/components/dropdown.mdx]: Ok, I'll see about changing the wording.

**Review by brandonlilly — APPROVED** [review]

**Pull request merged**